docs/metrics/README.md

```diff
@@ -127,27 +194,10 @@ each metric measurement. Refer to this
 
 ## Common issues that lead to missing metrics
 
-* The `Meter` used to create the instruments is not added to the
```

Comment:
need to keep the AddMeter missing part

Reply:
I added it back for now to control the scope of this PR.